_data/monthly-calls.yml

```diff
@@ -1,5 +1,21 @@
 videos:
-  - title: Conducting user research with people with disabilities
+  - title: Designing a repeatable research process for people with disabilities
+    subtitle: Conducting user research on how USWDS serves people with disabilities
```

Comment:
New subtitle: Managing and sustaining our user research program

@annepetersen, suggesting "managing," which strikes me as _slightly_ more plain language/approachable than conducting

Reply:
Updated the subtitle to "Managing and sustaining our user research". 

See the changes in this commit: [7586497](https://github.com/uswds/uswds-site/pull/2393/commits/7586497b8329d7c5c944ac49cd9b3eb2359b03c9)